### ISSUE TITLE
Investiger sur les effets de bord que peut avoir la suppresion d'un uttilisateur (en particulier dans le calcul des stats)

### ISSUE THREAD

**Issue by mxmeunier** [betagouv/api-subventions-asso#1427]

**Comment by mxmeunier** [comment]
On effectue les jointure de calcul de stats dans `UserAssociationVisitJoiner` à partir de la collection `users`. Il y donc un effet de bord à la suppression d'un utilisateur sur les statistiques liées car elles ne remonteront plus les informations de l'utilisateur supprimé (puisqu'on récupère en premier lieu la liste des utilisateurs).

Est-ce que la solution "quick win" et peut-être la meilleure si on souhaite garder les statistiques identiques après suppression d'un utilisateur, ne serait pas de ne pas supprimer l'utilisateur mais plutôt de le mettre à jour avec des champs `null` et simplement garder son `_id` ? En gros rendre l'utilisateur anonyme au lieu de le supprimer.

**Comment by alice-telescoop** [comment]
Oui, + un attribut `disabled` pour pouvoir facilement le sortir des stats du nombre d'utilisateurs

**Comment by alice-telescoop** [comment]
on crée un ticket ou on transforme ce ticket pour corriger ?

**Comment by mxmeunier** [comment]
J'ai modifié le ticket et je le prend :)
